contracts/samples/utils/OracleHelper.sol

```diff
@@ -67,7 +67,7 @@ abstract contract OracleHelper {
 
     function _setOracleConfiguration(
         OracleHelperConfig memory _oracleHelperConfig
-    ) internal {
+    ) private {
```

Comment:
leaving this method (called only from constructor) means that we still pay the SLOAD when using these parameters.
(and you can't make a member "immutable" if it is set from a private method)

Currently, the paymaster uses 15 slots (14, if we exclude "owner")
(checked with `solc --storage-layout`)
So it uses 14 storage slots on each operation, which is at minimum 14*2100 = 29400 gas .
But I suggest adding a "gascalc" test before optimizing it.